tutorials/prepare_dataset.md

```diff
@@ -45,9 +44,25 @@ python scripts/prepare_alpaca.py \
  --checkpoint_dir checkpoints/tiiuae/falcon-7b
 ```
 
+#### Truncating datasets
```

Comment:
Can you link to this section from the oom.md page?

Reply:
Good idea. Fixed that!